Getting-Started/Data/Defining-content/index.md

```diff
@@ -22,6 +22,12 @@ Go to the __Settings__ section in the backoffice. On the __Document Types__ node
 ![Creating a Document Type](images/Document-Type-Create.jpg)
 _You can also choose to create a Document Type without a template and create folders to organize your Document Types._
 
+You can also use compositions to create a new document type. Compositions allows you to inhert properties from other tabs. When using a mixed setup, you can take advantage of nesting and use compositions by visiting the Structure tab. A checklist like this should appear:
```

Comment:
inherit? or insert? but not inhert?

Reply:
I am unsure what you mean by this. Please clarify.

Reply:
```suggestion
You can also use compositions to create a new document type. Compositions allows you to inherit properties from other tabs. When using a mixed setup, you can take advantage of nesting and use compositions by visiting the Structure tab. A checklist like this should appear:
```

Reply:
Oh ok, thanks.